store/options.go

```diff
@@ -33,9 +33,9 @@ type Parameters struct {
 // DefaultParameters returns the default params to configure the store.
 func DefaultParameters() Parameters {
 	return Parameters{
-		StoreCacheSize: 4096,
+		StoreCacheSize: 512,
```

Comment:
We can say this is the default for LNs

Reply:
and other node type may have more by default.

Reply:
no concept of LNs in go-header - we can modify default config for BN + FN in nodebuilder if you want to increase nums